deps/external_deps.json

```diff
@@ -40,14 +40,9 @@
         "platform": ["el6", "el7", "fc23", "fc24"]
     },
     {
-        "name": "python-billiard",
-        "version": "3.3.0.17-2",
-        "platform": ["el6", "el7"]
```

Comment:
python-billiard >= 3.3.0.19 is required for celery 3.1.17.
3.3.0.20 is in EPEL7.